backend/bamboo/schemas/site.py

```diff
@@ -0,0 +1,13 @@
+from apiflask import Schema
+from apiflask.fields import Dict, String
+
+
+class SiteIn(Schema):
+    name = String(required=True)
+    config = Dict(required=True)
```

Comment:
Only name is required.

In the create form users should be able to offer other fields as well

Reply:
Fixed!